webextensions/api/windows.json

```diff
@@ -672,10 +672,12 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/windows/create",
             "support": {
               "chrome": {
-                "version_added": true,
+                "version_added": "≤44",
                 "notes": "<code>top</code>, <code>bottom</code>, <code>left</code>, <code>right</code> bounds must be at least 50% within visible screen space if specified, otherwise an error is thrown."
               },
-              "edge": "mirror",
+              "edge": {
```

Comment:
@queengooborg Where did this come from?

Reply:
This came from the version numbers present in downstream features!